src/model/User.ts

```diff
@@ -24,8 +25,8 @@ export interface User extends Document {
     verifyCode: string;
     verifyCodeExpiry: Date;
     isVerified: boolean;
-    isAcceptingMessages: boolean; // Consistent naming
-    messages: Message[];
+    isAcceptingMessages: boolean;
+    messages: Schema.Types.ObjectId[];
```

Comment:
you can not change the type of message schema to an ObjectId.

Reply:
done

Reply:
done